### ISSUE TITLE
No Speech Feedback when you respell a note

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#30483]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1.  Place focus on a note with an accidental, e.g. C#.
2. Press J to respell.
Notice the note changes to Db but there is no announcement from the screen reader.  The user has to arrow off and on again to hear the change has in fact taken effect.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.1

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
